spec/requests/experiment_spec.rb

```diff
@@ -43,13 +43,17 @@
 
   context 'UPDATE /experiments' do
     let(:experiment) { FactoryGirl.create(:experiment, user_id: token.resource_owner_id) }
+    let!(:samples) do
```

Comment:
You don't need to `let!` here, a regular `let` is fine. Even though `let` is evaluated lazily, it gets evaluated as part of the parameters that are required for sending the request (therefore, before the API actually executes any queries). `let!` is useful if you do not invoke the fixture before the API hits the database.

Otherwise, no it doesn't matter!